src/main/java/gregtech/common/pipelike/fluidpipe/tile/TileEntityFluidPipeTickable.java

```diff
@@ -121,16 +125,32 @@ private void distributeFluid(int channel, FluidTank tank, FluidStack fluid) {
 
             IFluidHandler pipeTank = tank;
             CoverBehavior cover = getCoverableImplementation().getCoverAtSide(facing);
-            if (cover != null) {
+
+            if (cover == null)
+                cover = getOtherCoverAt(facing, oppositeSide);
+            else {
```

Comment:
Does the capability check need to also check the cover returned by `getOtherCoverAt()`? If so, I believe this code should be replaced with my suggestion:

```suggestion
            if (cover == null) {
                cover = getOtherCoverAt(facing, oppositeSide);
            }
            if (cover != null) {
```

Reply:
resolved with https://github.com/GregTechCEu/GregTech/pull/1512/commits/91fda3e20db54d2f8427243ae4739957131b4e94